Craig_Campbell/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "greet",
+  "version": "0.1.0",
+  "description": "greet someone by name and time of day",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "./node_modules/mocha/bin/mocha test",
```

Comment:
I think Tyler wants us to be able to run "npm test" and have npm do the jshint and mocha tasks, via gulp - how would you modify your test script to handle that?